src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -6490,6 +6490,156 @@ static void Main()
             CompileAndVerify(source, expectedOutput: "1").VerifyDiagnostics();
         }
 
+        [Theory, WorkItem("https://github.com/dotnet/roslyn/issues/71333")]
+        //[InlineData(CSharp.LanguageVersion.Preview)]
```

Comment:
It's okay to include `Preview` already, even though it's technically redundant with `CSharpNext` for now. Let's uncomment
Same applies below

Reply:
When I tried that I think I got an xunit analyzer warning (that resulted in CI fail) because those inline data are duplicate.

Reply:
Argh. That's troublesome. The CSharpNext -> CSharp13 replacement is meant to be very mechanical. There's a risk we'd miss this comment. Maybe using a loop instead of a theory would be safer?